taxi/urls.py

```diff
@@ -16,8 +22,30 @@
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
+    path(
+        "manufacturers/manufacturer-create/",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create",
+    ),
+    path(
+        "manufacturers/<int:pk>/manufacturer-update/",
```

Comment:
The URL path 'manufacturers/<int:pk>/manufacturer-update/' should use hyphens instead of underscores. Consider changing it to 'manufacturers/<int:pk>/manufacturer-update/'.